cluster/gce/win1803/k8s-node-setup.psm1

```diff
@@ -54,17 +66,37 @@ function Log-Output {
     [switch]$Fatal
   )
   # TODO(pjh): what's correct, Write-Output or Write-Host??
-  Write-Output "${Message}"
+  Write-Host "${Message}"
   if (${Fatal}) {
     Exit 1
   }
 }
 
+# Checks if a file already exists and emits a message if it does.
+#
+# Returns $true if the file exists and the caller should NOT overwrite it.
+# Returns $false if the file does not exist, or if it does but the global
+# $REDO_STEPS variable is set to $true.
+function Test_FileExists {
```

Comment:
This might read easier if it was named something like `Should_Write` or `Confirm-Settable` (the latter trying to use "approved verbs"... :/ ), and return `$true` to write and `$false` to not write.

Reply:
Good suggestion, changed to "ShouldWrite_File" (in the verb-object style).

I agree that conforming to the approved verbs seems painful. Maybe we'll do that globally later but I'm not bothering right now.